anagrafica/permessi/funzioni.py

```diff
@@ -83,6 +91,7 @@ def permessi_ufficio_soci_unita(sede):
     """
     sede_qs = sede.queryset_modello()
     return [
+        (RUBRICA_UFFICIO_SOCI_UNITA, sede.espandi(includi_me=True, pubblici=True)),
         (GESTIONE_SOCI,     sede_qs),
         (ELENCHI_SOCI,      sede_qs),
     ]
```

Comment:
Direi che questo e' il punto migliore per inserire i permessi per tutte le rubriche di obiettivo. Questa funzione viene infatti usata sia dall'Ufficio Soci, che dal Presidente (vedi https://github.com/CroceRossaItaliana/jorvik/pull/405/files#diff-35972b881db257aec40a32f5bed266d6R78). Entrambi devono avere a disposizione tutte le rubriche per le proprie sedi, e sottostanti.

Idealmente una funzione privata puo' essere usata per riassumere tutte le rubriche degli obiettivi, in modo da essere usata sia qui che [qui](https://github.com/CroceRossaItaliana/jorvik/pull/405/files#diff-35972b881db257aec40a32f5bed266d6R113). Puo' essere unita alla fine del return con un bel `+` in modo simile a quel che accade sopra.

Questo dovrebbe risolvere quanto riscontrato da @luca-dex in https://github.com/CroceRossaItaliana/jorvik/pull/405#issuecomment-270720389.